test/integration/logs/benchmark/load_test.go

```diff
@@ -152,7 +152,7 @@ if [ $count -eq 2 ]; then
 	exit 0
 fi
 while true; do sleep 1; done
-],Args:[],WorkingDir:,Ports:[]ContainerPort{},Env:[]EnvVar{},Resources:ResourceRequirements{Limits:ResourceList{},Requests:ResourceList{},Claims:[]ResourceClaim{},},VolumeMounts:[]VolumeMount{},LivenessProbe:nil,ReadinessProbe:nil,Lifecycle:nil,TerminationMessagePath:/dev/termination-log,ImagePullPolicy:,SecurityContext:nil,Stdin:false,StdinOnce:false,TTY:false,EnvFrom:[]EnvFromSource{},TerminationMessagePolicy:,VolumeDevices:[]VolumeDevice{},StartupProbe:nil,}]
+],Args:[],WorkingDir:,Ports:[]ContainerPort{},Env:[]EnvVar{},Resources:ResourceRequirements{Limits:ResourceList{},Requests:ResourceList{},Claims:[]ResourceClaim{},},VolumeMounts:[]VolumeMount{},LivenessProbe:nil,ReadinessProbe:nil,Lifecycle:nil,TerminationMessagePath:/dev/termination-log,ImagePullPolicy:,SecurityContext:nil,Stdin:false,StdinOnce:false,TTY:false,EnvFrom:[]EnvFromSource{},TerminationMessagePolicy:,VolumeDevices:[]VolumeDevice{},StartupProbe:nil,ResizePolicy:[]ContainerResizePolicy{},}]
```

Comment:
This is okay. When choosing `v1.Pod` as type to test with, my reasoning was that surely it won't get modified much - oh boy, have I been wrong about that... :sweat_drops: 

I'll replace this with some simpler struct.

Reply:
It gets modified quite a bit. Some more changes expected with sidecar container effort that Sergey is driving.

Reply:
My [response](https://github.com/kubernetes/kubernetes/pull/102884/files/0065170a0f19d4fa66ee66d5459fd4238d092978#r1115537826) to your [above test/e2e_node/ comment](https://github.com/kubernetes/kubernetes/pull/102884#discussion_r1115361008) is not showing up here. 
ICYMI, I have a beta target issue https://github.com/kubernetes/kubernetes/issues/111877 tracking this. My recommendation is to keep the current e2e tests for now (which does true e2e and has caught issues so we know it is quite effective), and add new tests under e2e_node to complement this test by adding tests there in addition to various negative test cases that are [planned for beta](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/1287-in-place-update-pod-resources#beta) in e2e_node framework and then port over the current set of tests.

Reply:
Looks good to me.